app/assets/stylesheets/govuk_publishing_components/components/govspeak/_stat-headline.scss

```diff
@@ -8,22 +8,10 @@
 .govspeak, // Legacy class name that's still used in some content items - needs to be kept until `.govspeak` is removed from the content items.
 .gem-c-govspeak {
   .stat-headline {
-    margin-bottom: govuk-spacing(3);
-    margin-top: govuk-spacing(3);
-
-    @include govuk-media-query($from: tablet) {
-      margin-bottom: govuk-spacing(4);
-      margin-top: govuk-spacing(4);
-    }
-
-    p {
-      margin: 0;
-      @include govuk-font($size: 19, $weight: bold);
```

Comment:
Text seems to have lost bold weight on mobile 
<img width="156" alt="image" src="https://github.com/user-attachments/assets/3e94a5af-19a2-489b-bff2-df4fc8db442f" />
<img width="156" alt="image" src="https://github.com/user-attachments/assets/b122bdd3-1d72-42eb-9b15-3f604658d0ca" />

Reply:
Whoops, think I got a bit over-zealous there (had disappeared on desktop too). Have fixed.

Reply:
No worries, thanks!